config/config.yaml

```diff
@@ -28,14 +28,32 @@
  algorithms:
         - name: "pathlinker"
           params:
-                include: true
+                include: false
                 directed: true
                 run1:
                     k: range(100,201,100)
 
-        - name: "omicsintegrator1"
+        - name: "local_neighborhood"
+          params:
+                include : false
+                directed : false
+
+        - name: "random_walk"
```

Comment:
Is `randomwalk` more consistent with our naming convention for Path Linker and Omics Integrator? I'm not sure, we can discuss.

Reply:
Yes, I think `randomwalk` makes more sense with the naming convention. I named it `random_walk` just because it's one of my personal coding habits. I will also make changes to the relevant files to match the naming convention of SPRAS.